packages/core/src/Stores/notification-store.js

```diff
@@ -891,7 +891,31 @@ export default class NotificationStore extends BaseStore {
                                   text: localize('Yes, increase my limits'),
                               }
                             : {
-                                  route: routes.cashier_p2p_profile,
+                                  onClick: () => {
+                                      // TODO: Reroute this to my profile properly during p2p routing refactor
+                                      const routePromise = new Promise(resolve => {
+                                          // Need setTimeout as routing to p2p has a delay on mobile
+                                          if (window.location.pathname.includes(routes.cashier)) {
```

Comment:
I had to check if the current location is cashier as on mobile, the loading time to go to p2p is quicker than it is from outside of cashier. When routing to p2p from outside, the process is longer because it must route to cashier first, then to p2p.